internal/command/machine/status.go

```diff
@@ -87,14 +87,17 @@ func runMachineStatus(ctx context.Context) (err error) {
 			machine.InstanceID,
 			machine.State,
 			machine.FullImageRef(),
+			strings.Join(machine.Config.Init.Cmd, " "),
```

Comment:
why the Command and not the Entrypoint? should it be the last column as it sizes is the most variable one?

Reply:
I dunno. I figured most people would not be modifying the entrypoint, and can view the whole config with `-d`.